src/assignments/11-higher-order-function.js

```diff
@@ -32,3 +32,16 @@ const users = [
 ];
 
 // Program 1: create a function and print the user details whose age>=30 (use filter and map)
+/* eslint-disable */
+
+
+users.map((item, index)=> {
+  if(item.age >= 30){
+   console.log(item)
+  }
+})
+
+
+const Filter = users.filter(user => user.age >= 30)
```

Comment:
In case you wanna use filter and map in one line. Here it is.
Filter would get the object in the array that having age more or equals 30. It means after this method, we have a new array only contain people with target age. Then using map to get the attributes of the user.
```
const usersTarget = (users) => { const userOverThirty = users.filter(user => {return user.age >= 30;}).map(user =>({
 		id: user.id,
 		name: user.name,
 		country: user.country
 	}));
 	return userOverThirty;
}
```